app/controllers/products_controller.rb

```diff
@@ -55,8 +66,14 @@ def product_params
       :shipping_timeline_estimate,
       :origin_story,
       :submitted_page,
-      va_employees_attributes: [:id, :name, :role, :_destroy]
-      )
+      va_employees_attributes: [:id, :name, :role, :_destroy],
+    )
+  end
+
+  def multimedia_params
+    params.require(:practice).permit(
+      practice_multimedia_attributes: permitted_dynamic_keys(params[:practice][:practice_multimedia_attributes])
+    )
```

Comment:
I left the multimedia params as being nested under the `:practice` key, this is because the nested forms that are shared directly with the practice editor are a bit tough to untangle and making them dynamic depending on parent type will take extra time (although I'm sure it is doable). I'm on the fence about how egregious this is..